__tests__/redux-persist.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+const path = require('path');
+const assert = require('yeoman-assert');
+const helpers = require('yeoman-test');
+
+const storeCode = `\
+import { createStore } from 'redux';
+
```

Comment:
white space

Reply:
Intentional, between third and first party imports. Kinda pythonesque :P

Reply:
@rikvanderwerf